snmp/tests/test_e2e_core_vs_python.py

```diff
@@ -587,6 +603,9 @@ def assert_python_vs_core(
         else:
             aggregator.assert_metric(name, metric_type=mtype, tags=tags, count=count)
 
+    # Assert all metrics except the ones that are skipped.
+    for metric_name in metrics_to_skip:
+        aggregator.assert_metric(metric_name, at_least=0)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏I'm not sure to understand why this is needed since the metrics have already been excluded before from the aggregator:

https://github.com/DataDog/integrations-core/blob/86d847ded963fa5800dd4c6decdb2915beab4765/snmp/tests/test_e2e_core_vs_python.py#L579-L580

https://github.com/DataDog/integrations-core/blob/86d847ded963fa5800dd4c6decdb2915beab4765/snmp/tests/test_e2e_core_vs_python.py#L558-L559

Reply:
Good call, it's not necessary anymore indeed